zebrad/tests/acceptance.rs

```diff
@@ -2206,3 +2206,34 @@ async fn get_block_template() -> Result<()> {
 async fn submit_block() -> Result<()> {
     common::get_block_template_rpcs::submit_block::run().await
 }
+
+/// Check that the the end oif support code is called.
+#[test]
+fn end_of_support_is_checked() -> Result<()> {
```

Comment:
This test do not work as is, the idea was a simple test to see if the eos code is called. I didn't had enough time to fix it but i suppose it should be something simple missing.